src/node/rpc/call_types.h

```diff
@@ -155,4 +155,26 @@ namespace ccf
       bool valid = false;
     };
   };
+
+  struct GetRecoveryShare
```

Comment:
I believe the `submit_recovery_share.sh` script will need updating.

Reply:
Ah, that means semantically what you get and what you send is not the same actually. One is encrypted, one is plaintext. I'll make the fields `"encrypted_share"` and `"share"`, maybe?

Reply:
Yes, that seems even better!

Reply:
Done.